added footer, fully responsive on all devices

VladimirtheGreatest opened the pull request:
Footer, if you see some errors let me know. Tested it on mobile devices.

VladimirtheGreatest committed:
added footer, fully responsive on all devices

sasigit7 reviewed (changes requested):
Hey @VladimirtheGreatest , Here are the errors I see in your HTML code. 
1. Line 24/25 => duplicate head tag
2. Line 112 => duplicate data-index="5" and "a" tag is missing href attribute ?
3. Line 164 => I think width and height should be just a digit and shouldn't be in "px". Also an extra = to width property.  
4. Line 90, 95, 100, 105, 110, 115 and 119 => Remove all </a> end tags plz.  

I will have a look at your CSS later.

VladimirtheGreatest commented:
Thanks, deleted all data-indexes we dont need them anyway it was just automatically created by emmet abbreviation, deleted duplicate head tag, and removed all end tags
